core/includes/install.core.inc

```diff
@@ -1454,6 +1451,351 @@ function install_select_language_form($form, &$form_state, $files) {
   return $form;
 }
 
+/**
+ * Provide a list of available translation files.
+ *
+ * @return array
+ *   An array of language codes.
+ */
+function install_available_translations() {
```

Comment:
Should we name this newly-introduced function `install_get_available_translations()` instead (or something similar to that anyway)? As it is, it implies that it installs available translations.

Reply:
This function hopefully will be removed by the time we merge this PR, since we'll have an endpoint on localization to get a live list of available translations. But changed anyway.